src/glass/src/app/pages/welcome-page/welcome-page.component.html

```diff
@@ -0,0 +1,75 @@
+<div class="glass-welcome-page"
+     fxLayout="column"
+     fxLayoutAlign="center center">
+  <div class="card">
+    <div class="card-header">
+      <div class="card-title"
```

Comment:
```suggestion
      <div class="card-title"
```
```suggestion
      <div *ngIf="!checked"
           class="card-title"
           translate>
        Welcome to Aquarium
      </div>
      <div *ngIf="checked"
           class="card-title"
           translate>
        Requirements report
      </div>
```

Reply:
I'm not sure if I understand these two suggestions correctly. Do you want me to put the `*ngIf` first?

Reply:
Ah... no, you want me to change the title.